File: Lv.2/최솟값 만들기.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;
bool compare(int a,int b){
    return a>b;
}
int solution(vector<int> A, vector<int> B)
{
    int answer = 0;
    sort(A.begin(),A.end());
    sort(B.begin(),B.end(),compare);
    for(int i=0;i<A.size();i++){
        answer+= A[i]*B[i];
    }
    return answer;
}

File: Lv.2/최솟값 만들기.js
function solution(A,B){
    let answer = 0;
    
    A.sort((a,b)=>{
        return a-b;
    });
    B.sort((a,b)=>{
        return b-a;  
    })
    
    for(let i=0;i<A.length;i++){
        answer += A[i]*B[i];
    }
    return answer;
}